app/services/plum_event_processor/null_processor.rb

```diff
@@ -0,0 +1,7 @@
+class PlumEventProcessor
+  class NullProcessor
+    def self.process
```

Comment:
Dropping and logging seems like the right thing to do -- wouldn't want them to pile up and get processed later.